presto-main/src/main/java/com/facebook/presto/sql/planner/plan/JoinNode.java

```diff
@@ -29,8 +29,8 @@
 import java.util.List;
```

Comment:
I would use `Do not reshuffle JoinNode output symbols` for cmt msg

Reply:
Okay will update that